### PR TITLE
Bl-12491 publish tab UI fixes

### PR DESCRIPTION
**Pull request by nabalone**
This change is [Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6136)

### CODE REVIEW

File: src/BloomExe/Publish/Video/PublishAudioVideoAPI.cs [hunk 1]
```diff
@@ -391,8 +391,8 @@ public void AbortMakingVideo()
 		{
 			if (_recordVideoWindow != null)
 			{
-				_recordVideoWindow.Close();
 				_recordVideoWindow.Cleanup();
+				_recordVideoWindow.Close();
```
Comment (nabalone) [inline src/BloomExe/Publish/Video/PublishAudioVideoAPI.cs:395]: I think this was a bug that has been there for at least several versions. Null reference exception if you close it first and then try to clean it up

**Review by nabalone — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6136)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @andrew-polk)

___
*[`src/BloomExe/Publish/Video/PublishAudioVideoAPI.cs` line 237 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6136#-NgK5YMs-BvHvgLF5y2a:-NgKH5LJ5Y8Ff3xJiKM2:b-896fix) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/1a090ef76081cad41ee508015f9694141d84cb68/src/BloomExe/Publish/Video/PublishAudioVideoAPI.cs#L237)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

If this needs to be true, it is probably worth a comment as it isn't obvious why.
</blockquote></details>

Done.

**Review by andrew-polk — APPROVED** [review]
> Reviewed 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6136)* status: all files reviewed, 1 unresolved discussion

**Review by andrew-polk — APPROVED** [review]
> Reviewed 2 of 2 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/6136)* status: all files reviewed, 1 unresolved discussion

**Pull request merged**